core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1110,7 +1110,7 @@ protected function generateGlobalButtons()
 				$href = $this->addToUrl($v['href'] ?? '');
 			}
 
-			$return .= '<a href="' . $href . '" class="' . $v['class'] . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
+			$return .= '<a href="' . $href . '" class="' . ($v['class'] ?? '') . '" title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
```

Comment:
Shouldn‘t we omit the empty `class=""` attribute in this case as well?

Reply:
we could, but it would be more overhead so I thought we'd just keep it simple here 😅

Reply:
```suggestion
			$return .= '<a href="' . $href . '"' . (isset($v['class']) ? ' class="' . $v['class'] . '"' : '') . ' title="' . StringUtil::specialchars($title) . '"' . $attributes . '>' . $label . '</a> ';
```

Doesn‘t seem like too much overhead to me. 😉